<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>0</y>
        <width>371</width>
        <height>700</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="loadButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>341</width>
         <height>27</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Загрузить</string>
       </property>
      </widget>
      <widget class="QPushButton" name="addButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>171</width>
         <height>27</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Добавить камеру</string>
       </property>
      </widget>
      <widget class="QPushButton" name="deleteButton">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>160</y>
         <width>161</width>
         <height>27</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Удалить камеру</string>
       </property>
      </widget>
      <widget class="QComboBox" name="modelBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>341</width>
         <height>27</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QComboBox" name="cameraBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>341</width>
         <height>27</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QPushButton" name="moveButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>390</y>
         <width>341</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Сдвинуть</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="dxBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>360</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="minimum">
        <double>-400.000000000000000</double>
       </property>
       <property name="maximum">
        <double>400.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="dyBox">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>360</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="minimum">
        <double>-400.000000000000000</double>
       </property>
       <property name="maximum">
        <double>400.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="dzBox">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>360</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="minimum">
        <double>-400.000000000000000</double>
       </property>
       <property name="maximum">
        <double>400.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>340</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>dX</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>340</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>dY</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>340</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>dZ</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>450</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>kY</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="kyBox">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>470</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="kxBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>470</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>450</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>kX</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>450</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>kZ</string>
       </property>
      </widget>
      <widget class="QPushButton" name="scaleButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>500</y>
         <width>341</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Масштабировать</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="kzBox">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>470</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QPushButton" name="rotateButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>600</y>
         <width>341</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Повернуть</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>550</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>aZ</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="ayBox">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>570</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>359.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>550</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>aY</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="azBox">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>570</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>359.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>550</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>aX</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="axBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>570</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>359.000000000000000</double>
       </property>
      </widget>
      <widget class="QPushButton" name="rotateCButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>640</y>
         <width>341</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Повернуть камеру</string>
       </property>
      </widget>
     </widget>
     <widget class="QGraphicsView" name="Canvas">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>700</width>
        <height>700</height>
       </rect>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
